shortcircuit-api/src/test/java/com/powsybl/shortcircuit/ShortCircuitParametersTest.java

```diff
@@ -278,7 +278,35 @@ void readVersion12() {
         assertTrue(parameters.isWithFeederResult());
         assertEquals(StudyType.SUB_TRANSIENT, parameters.getStudyType());
         assertEquals(0, parameters.getMinVoltageDropProportionalThreshold(), 0);
-        assertEquals(0.8, parameters.getSubTransientCoefficient(), 0);
+        assertEquals(0.7, parameters.getSubTransientCoefficient(), 0);
+        assertFalse(parameters.isWithLoads());
+        assertFalse(parameters.isWithShuntCompensators());
+        assertFalse(parameters.isWithVSCConverterStations());
+        assertTrue(parameters.isWithNeutralPosition());
+        assertEquals(InitialVoltageProfileMode.CONFIGURED, parameters.getInitialVoltageProfileMode());
+        List<VoltageRange> voltageRanges = parameters.getVoltageRanges();
+        assertEquals(3, voltageRanges.size());
+        assertEquals(1.05, voltageRanges.get(0).getRangeCoefficient());
+        assertEquals(Range.between(380., 410.), voltageRanges.get(0).getRange());
+        assertEquals(1.1, voltageRanges.get(1).getRangeCoefficient());
+        assertEquals(Range.between(0., 225.), voltageRanges.get(1).getRange());
+        assertEquals(1.09, voltageRanges.get(2).getRangeCoefficient());
+        assertEquals(Range.between(230., 375.), voltageRanges.get(2).getRange());
```

Comment:
The default value can be tested, for instance here:
```suggestion
        assertEquals(Range.between(230., 375.), voltageRanges.get(2).getRange());
        assertTrue(parameters.isDetailedReport());
```

Reply:
Actually this is just an improvement of the test of version 1.2 that does not have this new parameter. The new parameters introduced in this version were not introduced so I added them, like that the test make more sense, testing all the new parameters. 
To test `detailedReport`, I created a test that is right below

Reply:
I thought that the default value of the parameter wasn't covered by any test. But in fact, `writeExtension()` will fail if the default value is changed. 
=> No need for another test.